chain/client/src/stateless_validation/chunk_validator/mod.rs

```diff
@@ -742,14 +764,6 @@ impl Client {
             return Err(Error::NotAChunkValidator);
         }
 
-        if !self.epoch_manager.verify_chunk_state_witness_signature(
```

Comment:
you are removing this from here because signature verification will happen for each part right?

Reply:
Yep! The signature verification would happen for each part now. It'll be part of a follow up PR.

Reply:
To maintain compatibility with the current message, I have the `process_signed_chunk_state_witness` which would eventually go away.